src/features/pages/404-error/content/not-found.module.scss

```diff
@@ -0,0 +1,20 @@
+@use 'features/styles/theme/theme-mixins' as *;
+
+.not_found {
+    padding-top: 12rem;
```

Comment:
Hi @aswathy-deriv can replace this with logical property 👀 
Thank you  🙏

Reply:
sure @fasihali-deriv